### DISCUSSION TITLE
Welcome to test-microservice-frameworks Discussions!

### DISCUSSION THREAD

**Discussion by ozkanpakdil** [ozkanpakdil/test-microservice-frameworks#719] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ozkanpakdil** [comment]
I just added .net8 to the testing list. you can see it on the graph
![image](https://github.com/ozkanpakdil/test-microservice-frameworks/assets/604405/c0849e35-2224-4b66-904f-c29e29272b99)

I am increasing the width of the graph, fingers crossed it will look better.

**Comment by hollandjake** [comment]
Any thoughts on increasing the resolution of tracking as bun and rust both max out the rps? 

Also note the rps is calculated wrong as you are using a scaling number of users so the average is mis calculated

**Reply by ozkanpakdil** [reply]
You want to use k6 for proper tracking, said in this [message](https://github.com/ozkanpakdil/test-microservice-frameworks/pull/915#issuecomment-3767111287) too, I dont see any blocker. I may do it next weekend or later. I have not used k6 before so it will take sometime for me to understand and adopt. 

About wrong averages, it's gatling responsibility, so I am getting the `mean response` from the gatling result, we can use `99 percent` as the average here ? or do you think these numbers are totally wrong, if that is the case I do not know what else to do, I may try to move to jmeter, but again it will take some weekends :) feel free to open a PR if you know k6 and easy to adopt. 

My only suggestion would be lets have gatling with k6 togather for a while, to see the differences. maybe in the future we can drop gatling totally. 

<img width="800" height="436" alt="image" src="https://github.com/user-attachments/assets/8dab1b0a-f569-47e5-a487-546464abbcdb" />

**Reply by hollandjake** [reply]
I'd be happy to whip up a PR for you
